src/components/Frame/components/Loading/Loading.scss

```diff
@@ -14,7 +14,7 @@ $height: rem(3px);
   height: 100%;
   transform-origin: 0;
   background-color: var(--p-icon-success, color('teal'));
-  transition: transform duration(slowest) easing(out);
+  transition: transform duration(slowest) easing(out) 15ms;
```

Comment:
I actually think the default `linear` feels more natural here.

This `delay` seems a bit magical. Is it an arbitrary number because it _just happens to work nicely_, or is it informed by _something_?

Reply:
The delay came from the previous `setTimeout` so it is definitely magicial.

Reply:
Just in case you maybe accidentally changed the wrong thing:

My comment was to change the `timing-function` to be `linear`. Your latest commit made a change to the `duration` - which is maybe fine, I didn't test the speed difference.